modules/xfem/src/reporters/CrackGrowthReporterBase.C

```diff
@@ -0,0 +1,85 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "CrackGrowthReporterBase.h"
+#include "CrackMeshCut3DUserObject.h"
+#include "VectorPostprocessorInterface.h"
+#include "MathUtils.h"
+
+InputParameters
+CrackGrowthReporterBase::validParams()
+{
+  InputParameters params = GeneralReporter::validParams();
+  params.addRequiredParam<UserObjectName>("crackMeshCut3DUserObject_name",
+                                          "The CrackMeshCut3DUserObject user object name");
+  params.addRequiredParam<Real>(
+      "max_growth_size",
```

Comment:
I would rename this to `max_growth_increment` because it's more precise. `size` could imply a vector size or something.